x-pack/plugins/actions/server/builtin_action_types/email.test.ts

```diff
@@ -396,4 +396,37 @@ describe('execute()', () => {
       }
     `);
   });
+
+  test('renders parameter templates as expected', async () => {
```

Comment:
This is the first of a new set of tests for action types with optional `renderParameterTemplates()` methods.  They don't exhaustively test the escaping, as that is done in the render-specific code - it only tests that the escaping is basically happening as expected.